infra/config/example.ts

```diff
@@ -36,5 +36,14 @@ export const config: EnvConfig = {
     host: "myhealthapp.com",
   },
   systemRootOID: "2.16.840.1.113883.3.999999",
+  cwOrgName: "Test",
+  cwMemberOid: "1.1.1.1.1.1.28.12.12.1234.123",
+  cwOrgManagementKey: "-----BEGIN PRIVATE KEY-----example-----END PRIVATE KEY-----",
+  cwOrgManagementCertification: "-----BEGIN CERTIFICATE-----example-----END CERTIFICATE-----",
+  cwOrgMemberKey: "-----BEGIN PRIVATE KEY-----example-----END PRIVATE KEY-----",
+  cwOrgMemberCertificate: "-----BEGIN CERTIFICATE-----example-----END CERTIFICATE-----",
```

Comment:
I think I missed this in the previous PR, but we'll want to put these configs in secrets, just like we do with our provider client ids/secrets.

Essentially we never want to find ourselves committing a secret to a code base - even if it's internal

Reply:
^ ofc will need to update the infra code accordingly